.github/workflows/deploy-nginx.yml

```diff
@@ -25,37 +25,70 @@ jobs:
         id: short_sha
         run: echo "short_sha=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
 
+      - name: Notify Build And Deploy Start
+        id: build_message
+        uses: ./.github/actions/slack-notify
+        with:
+          step: "start"
+          component: "http-proxy"
+          image_tag: ${{ steps.short_sha.outputs.short_sha }}
+          channel: ${{ secrets.SLACK_CHANNEL_ID }}
+          slack_token: ${{ secrets.SLACK_BOT_TOKEN }}
+
       - name: "Authenticate with Google Cloud"
         uses: "google-github-actions/auth@v1"
         with:
           credentials_json: "${{ secrets.GCLOUD_SA_KEY }}"
 
-      - name: "Set up Cloud SDK"
-        uses: "google-github-actions/setup-gcloud@v1"
-
-      - name: Install gke-gcloud-auth-plugin
-        run: |
-          gcloud components install gke-gcloud-auth-plugin
-
-      - name: Setup kubectl
-        run: |
-          gcloud container clusters get-credentials dust-kube-private --region us-central1
-
       - name: Build the image on Cloud Build
         run: |
           mkdir empty
           cp ./nginx.Dockerfile empty
           cd empty
           chmod +x ../k8s/cloud-build.sh
-          ../k8s/cloud-build.sh --image-name=nginx-proxy --dockerfile-path=./nginx.Dockerfile --working-dir=./ --gcloud-ignore-file=../.gcloudignore-nginx --dust-client-facing-url=https://dust.tt
+          ../k8s/cloud-build.sh \
```

Comment:
This build is very weird 🤔  (cc @Fraggle if you can shed some lights?). Anyway with the new infrastructure setup this should probably be moved to `dust-infra`.

Reply:
What do you mean by "weird" ?

Reply:
Why do we need an empty directory?